File: wiz-event-mall-java/src/main/java/com/example/wizeventmalljava/domain/event/repository/EventWinnerRepository.java
package com.example.wizeventmalljava.domain.event.repository;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.example.wizeventmalljava.domain.event.entity.Event;
import com.example.wizeventmalljava.domain.event.entity.EventWinner;

@Repository
public interface EventWinnerRepository extends JpaRepository<EventWinner, Long> {

	boolean existsByEventIdAndMemberId(Long eventId, Long memberId);
	Long countByEvent(Event event);
}

File: wiz-event-mall/src/main/kotlin/com/example/wizeventmall/domain/event/repository/EventWinnerRepository.kt
package com.example.wizeventmall.domain.event.repository

import com.example.wizeventmall.domain.event.entity.EventWinner
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface EventWinnerRepository: JpaRepository<EventWinner, Long> {
}